features/LastSeen.feature.md

```diff
@@ -23,15 +31,12 @@ And the device `${fingerprint_deviceId}` publishes this message to the topic
 }
 ```
 
-<!-- @retry:delayExecution=5000 -->
-
 ## Retrieve last seen timestamp on connect
 
 Given I store `$fromMillis(${ts})` into `tsISO`
 
-When I connect to the websocket using fingerprint `${fingerprint}`
-
-<!-- @retry:tries=5,initialDelay=5000,delayFactor=1 -->
+When I connect to the websocket using fingerprint `${fingerprint}` in `5`
+seconds
```

Comment:
```suggestion
Given I wait for 5 seconds

When I connect to the websocket using fingerprint `${fingerprint}`
```

If user must wait 5 seconds before connect, add a separate wait step.

Reply:
But I wonder, why must the user wait now?

Reply:
Last seen will be recorded in DB in async way. So, sometimes when you disconnect and reconnect immediately, the DB is not done yet and the test is failed.